package.json

```diff
@@ -54,8 +54,8 @@
     "prettier": "^1.19.1",
     "pretty-format": "26.6.2",
     "raw-loader": "^0.5.1",
-    "react-dnd-test-backend": "^9.4.0",
-    "react-dnd-test-utils": "^9.4.0",
+    "react-dnd-test-backend": "^9.5.0",
+    "react-dnd-test-utils": "^9.5.0",
```

Comment:
Perhaps bump to version 14, like https://github.com/theforeman/foreman-js/commit/7282ad650c60ab4bc5ce3bdc6d665158026f0cb5 did?